src/utils/recording.ts

```diff
@@ -0,0 +1,19 @@
+import { MoujiRecording } from "../types/MoujiRecording";
+
+export function serializeRecording(recording: MoujiRecording): string {
+  return (
+    recording.meta.interval +
+    "||" +
+    recording.data.flatMap((pos) => `${pos.x},${pos.y}`).join("|")
+  );
+}
+export function deserializeRecording(recording: string): MoujiRecording {
+  const [header, records] = recording.split("||");
```

Comment:
Something to think about - if I give the wrong `-f` or the file gets afu i will get a weird error that is hard to understand as a human. Although, you might want to check the format of the file you're going to parse somewhere before you pass it in to this mf

Reply:
Yeah that's true. We should just use JSON lol. That way we can keep a recording version and stuff in there too.